superset-frontend/src/features/databases/DatabaseModal/index.tsx

```diff
@@ -1282,10 +1299,10 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
   }, [sshPrivateKeyPasswordNeeded]);
 
   useEffect(() => {
-    if (db && isSSHTunneling) {
-      setUseSSHTunneling(!isEmpty(db?.ssh_tunnel));
+    if (db?.parameters?.ssh !== undefined) {
```

Comment:
similar thing expected here? `undefined` is different than `false`?

Reply:
Yes just differentiating between 3 possible states, never checked, checked, unchecked